packages/app/src/PasswordResetPage.tsx

```diff
@@ -20,7 +20,7 @@ export default function PasswordResetPage({
     <div>
       <Form
         className="text-start"
-        action={`${API_SERVER_URL}/password-reset`}
+        action={`${SERVER_URL}/password-reset`}
```

Comment:
double check that this url won't be updated dynamically, right ?

Reply:
Oh this is true, though this isn't important in local mode...

Reply:
We'll likely want to wrap this in a context and pull it from there in the future, but that's out of scope for this PR for now.